sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -39,6 +39,19 @@ def _to_timestamp(date):
         timestamp += date.microsecond / 1e6
     return timestamp
 
+def _to_datetime(value):
```

Comment:
I think this should be given a better name, otherwise I don't know what is being converted into a datetime.